ci-operator/README.md

```diff
@@ -5,6 +5,26 @@ This document describes how to create CI jobs for Openshift components using
 ci-operator and is intended for component developers who want to add tests to
 their CI process.
 
+Conventions
+-----------
+
+Under this directory, we have three main directories:
+
+*  `config/$repo/$branch.json`
+   Contains your ci-operator definition which describes how the images and tests
+   in your repo works. These files are copied into config maps on the CI cluster
+   and referenced by Prow jobs.
+*  `jobs/$repo/(pull|branch)-$repo-(presubmit|postsubmit|periodic)-$branch.yaml`
```

Comment:
pull === presubmit
branch === postsubmit

why?

Reply:
also it's $org-$repo right now

Reply:
Ah, I was using $repo as $org/$repo.